website/blog/2022-07-28-srb-tapioca.md

```diff
@@ -116,6 +116,13 @@ Tapioca has some unique benefits compared to `srb rbi`:
   method stubs. It will also generate `T.let` annotations for constants defined
   by the gem.
 
+- Tapioca pulls community-driven type annotations for public gems from
+  [rbi-central](https://github.com/Shopify/rbi-central) instead of from
+  [sorbet-typed](https://github.com/sorbet/sorbet-typed), which `srb rbi` uses.
+  The difference between the two repos is largely just about how metadata is
+  stored. It should be painless to contribute any relevant `sorbet-typed` RBI
+  files to `rbi-central` if your codebase depended on them.
```

Comment:
There is also a difference on how we check the RBIs for correctness:

* Runtime checks, we ensure that every constant/method defined in the RBI actually exist at runtime
* Static checks: We ensure that 1) the definition is not redundant with what Tapioca produces (like adding an empty scope already generated by Tapioca, or a method without a signature) 2) doesn't conflict with what Tapioca generates (like changing the arity of a method) 3) that the generated RBI + the written one do not produce any error when running Sorbet on it.

Reply:
I was trying to figure out how much to say "yeah it's just better" + link to the full docs, versus how much to explain in detail about why it's better.

Feel free to push whatever changes you'd like to this section.

Reply:
Done in 36ae5d915 👍